packages/snaps-controllers/src/cronjob/CronjobController.ts

```diff
@@ -267,24 +332,139 @@ export class CronjobController extends BaseController<
   }
 
   /**
-   * Unregister all jobs related to the given snapId.
+   * Schedule a background event.
+   *
+   * @param backgroundEventWithoutId - Background event.
+   * @returns An id representing the background event.
+   */
+  scheduleBackgroundEvent(
+    backgroundEventWithoutId: Omit<BackgroundEvent, 'id' | 'scheduledAt'>,
+  ) {
+    // removing milliseond precision and converting to UTC.
+    const scheduledAt = DateTime.fromJSDate(new Date())
+      .toUTC()
+      .toFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
Why is this different from the way we format it in the RPC handler?

E.g. `.toISO({
        suppressMilliseconds: true
      })`